### PR TITLE
Crisis Plan

### PR DESCRIPTION
**Pull request by DanielYoung09**
Crisis plan added. FAB added to bottom nav bar for access. Data Storage created for crisis plan.

image
image

### CODE REVIEW

**Review by ShenY11 — APPROVED** [review]
Looks good to me!

File: lib/screens/CrisisPlan.dart [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+        body: ListView(
+          scrollDirection: Axis.vertical,
+          shrinkWrap: true,
+          padding: const EdgeInsets.only(left: 20.0, right: 20.0),
+          children: [
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                // IconButton(
```
Comment (AlexShor0103) [inline lib/screens/CrisisPlan.dart:100]: Is there any need to keep this block of commented out code?

File: pubspec.lock [hunk 2]
```diff
@@ -569,4 +569,4 @@ packages:
     version: "0.2.0+2"
 sdks:
   dart: ">=2.18.0 <3.0.0"
-  flutter: ">=3.0.0"
+  flutter: ">=2.10.0"
```
Comment (AlexShor0103) [inline pubspec.lock:572]: Should we go back a version? In the previous iteration this was set to 3.0.0

File: lib/screens/MapScreen.dart [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import 'dart:async';
 import 'dart:core';
-import 'package:google_maps_flutter_web/google_maps_flutter_web.dart';
+//import 'package:google_maps_flutter_web/google_maps_flutter_web.dart';
```
Comment (AlexShor0103) [inline lib/screens/MapScreen.dart]: Should we leave this commented out, delete it entirely, or bring it back?

**Review by AlexShor0103 — APPROVED** [review]
Overall great job! The major thing I noticed was a block of commented out code and us going back to flutter 2.10.0 which I wasn't sure was intentional. Other than that, it's solid!

File: pubspec.lock (hunk 2 above)
Comment (DanielYoung09) [reply pubspec.lock:572]: I'm not sure why or how that happened. I certainly don't remember reverting o.O

**Pull request merged**